src/pages/reviewPage.tsx

```diff
@@ -10,35 +10,125 @@ import Header from "../components/Header";
 import { signIn, useSession } from "next-auth/react";
 import Router, { useRouter } from "next/router";
 import Link from "next/link";
+import { Dialog, Transition } from "@headlessui/react";
+import { Rating } from "react-simple-star-rating";
+import { Box } from "@mui/material";
 const reviewPage: NextPage = () => {
```

Comment:
```suggestion
const ReviewPage: NextPage = () => {
```
React component (including pages) should be PascalCase